v2.2/debug-merge-logs.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Merge debug logs for all nodes
+summary: Learn the command for merging the collected debug logs from all nodes in your cluster.
+toc: true
+---
+
+The `debug merge-logs` [command](cockroach-commands.html) merges the logs for all nodes in the cluster collected using the [`debug zip`](https://www.cockroachlabs.com/docs/stable/debug-zip.html) command. This command helps in debugging cluster issues by making it easier to parse the logs for all nodes and locating an issue.
```

Comment:
I find that this paragraph overly emphasizes a relationship between merge-logs and zip. Maybe something like 

```
The `debug merge-logs` [command](cockroach-commands.html) merges log files from multiple nodes into a single time-ordered stream of messages with an added per-message prefix to indicate the corresponding node. It can be used in conjunction with logs collected using the [`debug zip`](https://www.cockroachlabs.com/docs/stable/debug-zip.html) command to aid in debugging.
```